internal/helm/controller/reconcile.go

```diff
@@ -65,7 +64,7 @@ const (
 // release changes are necessary, Reconcile will create or patch the underlying
 // resources to match the expected release manifest.
 
-func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
+func (r HelmOperatorReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
```

Comment:
@camilamacedo86 Same as in ansible, can we pass the same context used in reconcile in `updateResource` and other functions too which are called by reconciler?